xunit.runner.wpf/ViewModel/MainViewModel.cs

```diff
@@ -402,6 +429,22 @@ private void OnExecuteCancel()
             this.cancellationTokenSource.Cancel();
         }
 
+        private void OnTraitSelectionChanged()
+        {
+            this.searchQuery.TraitSet = new HashSet<TraitViewModel>(
+                this.traitCollectionView.Collection.Where(x => x.IsSelected),
+                TraitViewModelComparer.Instance);
+            FilterAfterDelay();
+        }
+
+        private void OnTraitsClear()
+        {
+            foreach (var cur in this.traitCollectionView.Collection)
+            {
+                cur.IsSelected = false;
+            }
+        }
```

Comment:
Don't believe so.  The clearing of the selection will trigger `OnTraitSelectionChanged` which will call `FilterAfterDelay`.  I tested this at some point last night.  Will verify I didn't later break it.